File: Scan_Converting_Sector_Trigonometric_Method.py

# Scan Converting Sector using Trigonometric Method
import numpy as np
import matplotlib.pyplot as plt

r = 50
theta = np.linspace(0, np.pi/3, 200)
x = np.concatenate(([0], r*np.cos(theta), [0]))
y = np.concatenate(([0], r*np.sin(theta), [0]))

plt.fill(x, y)
plt.gca().set_aspect('equal', 'box')
plt.title("Sector - Trigonometric Method")
plt.grid(True)
plt.show()

File: Scan_Converting_Sector_Trigonometric_Method.m

% Scan Converting Sector using Trigonometric Method
clc; clear; close all;
r = 50; theta = linspace(0, pi/3, 200);
x = [0 r*cos(theta) 0];
y = [0 r*sin(theta) 0];
fill(x,y,'c'); axis equal; grid on;
title('Sector - Trigonometric Method');
